docs/_static/switcher.json

```diff
@@ -0,0 +1,72 @@
+[
+  {
+    "name": "dev",
+    "version": "latest",
+    "url": "https://ctapipe.readthedocs.io/en/latest/"
+  },
+  {
+    "name": "v0.19.2 (stable)",
```

Comment:
0.19.2 is repeated, once with (stable) once without. Why?

Stable should be the latest release (0.19.3).  And probably not have a version tag, since we'd need to always update that...